src/components/wayfinder/wayfinder.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import data from "../../data/wayfinderData.js";
+
+export default class Wayfinder extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      usersPath: ["start"],
+      data: data
+    };
+  }
+
+  questionList = () => {
+    const { data, usersPath } = this.state;
+
+    return usersPath.map(item => {
+      return <QAContainer question={data[item]} key={data[item].question} />;
```

Comment:
I think 'question' key will create confusion - how about 'qablock' or just 'block'?

Reply:
Actually, could you not just pass in the data[item] as a rest operator, and destructur the questions and answer options in the QAContainer component? 
`return <QAContainer  key={data[item].question {...data[item]} } />;`